<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandConditions</class>
 <widget class="QDialog" name="DialogCommandConditions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Condition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Variables and Switches</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButtonSwitch">
             <property name="text">
              <string>Switch</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="WidgetVariableSwitch" name="widgetSwitch" native="true">
               <property name="enabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelSwitch">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>is</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxSwitch">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>ON</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>OFF</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButtonLocalSwitch">
             <property name="text">
              <string>Local switch</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="WidgetComboBoxLocalSwitches" name="widgetLocalSwitches" native="true">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelLocalSwitch">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>is</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxLocalSwitch">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>ON</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>OFF</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioButtonVariable">
             <property name="text">
              <string>Variable</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="WidgetVariableSwitch" name="widgetVariableVariable" native="true">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>175</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>175</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelVariable">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>is</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="2">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>68</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1">
            <widget class="WidgetComboBoxOperation" name="widgetVariableOperation" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>68</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="1">
            <widget class="PanelPrimitiveValue" name="widgetVariableVariableConstant" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Add &quot;else&quot; when conditions do not apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetVariableSwitch</class>
   <extends>QWidget</extends>
   <header>widgetvariableswitch.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetComboBoxOperation</class>
   <extends>QWidget</extends>
   <header>widgetcomboboxoperation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetComboBoxLocalSwitches</class>
   <extends>QWidget</extends>
   <header>widgetcomboboxlocalswitches.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandConditions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandConditions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
